am/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This file documents all notable changes to [Gravitee.io Access Management 3.x](h
 ### 1.0.51
 
 - [X] Add upgrader framework job
```

Comment:
Does this feat (upgrader fwk), added in 1.0.51 ? :thinking:

Reply:
https://github.com/gravitee-io/helm-charts/pull/262/files#r1050806000